components/automate-cs-nginx/habitat/config/nginx.conf

```diff
@@ -83,12 +83,21 @@ http {
     keepalive 8;
   }
 
+{{#if cfg.external_automate.enable}}
+  upstream external-automate {
+    server {{cfg.external_automate.endpoint}};
+    keepalive 8;
```

Comment:
I'm all for this for now but I'm curious to see how keepalives against an external automate will work in practice.